Libs/Mesh/Mesh.h

```diff
@@ -107,6 +113,12 @@ class Mesh
   /// number of faces
   vtkIdType numFaces() const { return mesh->GetNumberOfCells(); }
 
+  /// returns matrix with vertex data
```

Comment:
Can you please describe this more specifically? Even looking at the definition I still don't understand what it's doing.

Reply:
Done